DEVELOPMENT.md

```diff
@@ -136,31 +136,32 @@ Kubernetes cluster in your designated environment, if necessary.
 ### Deploy Istio
```

Comment:
Should we just point to "pick a networking layer" from https://knative.dev/docs/install/any-kubernetes-cluster/ ?

Reply:
My understanding is that the Istio installation instruction within https://knative.dev/docs/install/any-kubernetes-cluster/ is mainly for production installation with multiple options.

Here for development, we may want to provide a simple installation method for developers to quickly bootstrap.

Reply:
I'm +1 on keeping this simple, and it sounds like there isn't a simple Istio installation option anymore.  "Clone _another_ repo and run some bash" really sucks, and I think we need a higher bar than that. 😞 

FWIW, I haven't run Istio anywhere in 6 months, and ran it side-by-side (barely used) for 3 months prior, and haven't had any real issues.  🤷

Reply:
To simply the installation, I can wrap those steps within a script (something like https://github.com/knative/serving/blob/master/test/e2e-networking-library.sh#L17) which hides the underneath execution details (e.g. clone a repo). WDYT?

Or do you suggest using other simpler network solution here?

Reply:
Have to say, Im also pretty +1 on using kourier or something for serving dev. Folks who actually need istio already know exactly how to install it and/or can follow instructions on the net-istio repo for cloning and installing. I can't think of a super strong reason to make people jump through all the hoops needed to get istio running locally, and it's going to be tested in CI anyway. Presumably kourier might also be a little lighter on minimum requirements for local dev, even compared to the "minimal" istio profile?

Reply:
Ack! +1 for defaulting to something that doesn't require an additional installation step.

Reply:
@ZhiminXiang IMO if it needs to involve bash, it's too complicated.

I can generally get a new cluster bootstrapped in 1-2 kubectl/ko commands, so it seems silly to me that this should be so complicated, and I want the barrier to contribution (at least kicking the tires) to be very low.

Reply:
Currently kourier test grid is a bit red https://testgrid.knative.dev/serving#kourier-stable (especially for scaling tests) comparing with istio-no-mesh. Would this be a concern?

Reply:
I switched to use kourier and refer Istio and contour from the development doc. Let me know what you think.